File: Tema 5 - Arrrays/ejercicio2/archives/automaticamente/automaticamente.php
<?php
if (isset($_GET["numerosAuto"])) {
  $numero = json_decode($_GET["numerosAuto"]);
  $mensaje = "";
  $contador = 0;
  $max;
  $min;

  $max = $numero[0];
  $min = $numero[0];

  foreach ($numero as $elemento) {
    if ($elemento > $max) {
      $max = $elemento;
    }
    if ($elemento < $min) {
      $min = $elemento;
    }
  }
  $mensaje = " <table border='1'><tr><th>Indice</th><th>Numero</th></tr>";
  $contador = 0;
  foreach ($numero as $elemento) {
    if ($max == $min) {
      $mensaje .= "<tr> <td>$contador</td> <td>$elemento es max y min </td> </tr>";
    } elseif ($elemento == $min) {
      $mensaje .= "<tr> <td>$contador</td> <td>$elemento minimo </td> </tr>";
    } elseif ($elemento == $max) {
      $mensaje .= "<tr> <td>$contador</td> <td>$elemento maximo </td> </tr>";
    } else {
      $mensaje .= "<tr> <td>$contador</td> <td>$elemento </td> </tr>";
    }
    $contador += 1;
  }
  $mensaje .= "</table>";
  echo $mensaje;
}

File: Tema 5 - Arrrays/ejercicio2/archives/automaticamente/automaticamente.py
"""
Este módulo muestra un MENSAJE incrementando el valor de i.
"""
import sys
import json

if len(sys.argv) > 1:
    NUMERO = json.loads(sys.argv[1])
    CONTADOR = 0
    MENSAJE = ""
    MAX = NUMERO[0]
    MIN = NUMERO[0]
    for elemento in NUMERO:
        if elemento > MAX:
            MAX = elemento
        if elemento < MIN:
            MIN = elemento
    MENSAJE += "<table border='1'><tr><th>Indice</th><th>Numero</th></tr>"
    for contador, valor_numero in enumerate(NUMERO):
        if MAX == MIN:
            MENSAJE += f"""<tr><td>{contador}</td><td>{valor_numero} max y min</td></tr>"""
        elif valor_numero == MAX:
            MENSAJE += f"""<tr><td>{contador}</td><td>{valor_numero} maximo</td></tr>"""
        elif valor_numero == MIN:
            MENSAJE += f"""<tr><td>{contador}</td><td>{valor_numero} minimo</td></tr>"""
        else:
            MENSAJE += f"""<tr><td>{contador}</td><td>{valor_numero}</td></tr>"""
    print(f"{MENSAJE}")
